### ISSUE TITLE
Draggable and editable gcode modifier

### ISSUE THREAD

**Issue by guestisp** [supermerill/Slic3r#202]
Currently, when you add a gcode modifier at a certain level (with the vertical "scoll bar"), it's impossible to move it up, down or change the gcode content.

The only solution is to delete that gcode modifier and create a new one from scratch.
A draggable feature would be great and also something to edit the content.

**Comment by supermerill** [comment]
There are a good chance that prusa will continue to improve it on their own. Unless they said they won't touch it anymore, i'll let them do it.

**Comment by liftbag** [comment]
@guestisp yes, you cannot move it up and down, but you can edit it by right click on the delete simbol (x).
